hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java

```diff
@@ -426,10 +426,10 @@ public void reset() {
     public void shutdown(final boolean terminate) {
         logger.finest("Shutting down services...");
         waitNotifyService.shutdown();
+        operationService.shutdown();
```

Comment:
Thanks, rerunned failing tests, they worked fine with that fix. 

Maybe one another thing to prevent unintended usage of operationService and to make that more visible, operationService can throw an exception if one tries to use it after its' shutdown. I will create an issue for this.